node/bundler.ts

```diff
@@ -49,6 +50,7 @@ const bundle = async (
     onBeforeDownload,
     systemLogger,
     internalSrcFolder,
+    bootstrapURL = 'https://edge.netlify.com/bootstrap/index-combined.ts',
```

Comment:
My gut feeling is that since we're only using this for the ISC check, it's not of the utmost importance that this comes from DevCycle. So I made it optional for now, and i'll open an issue for us to have buildbot pass this in. Once that's there, we can make it required here.